pkg/c25519/Makefile

```diff
@@ -0,0 +1,36 @@
+PKG_NAME    = c25519
+PKG_URL     = https://www.dlbeer.co.nz/downloads
+PKG_VERSION = 2017-10-05
+PKG_EXT     = zip
+PKG_LICENSE = PD
+PKG_SHA512  = dbfb4285837ab2ea3d99c448b22877cc7a139ccbaebb1de367e2bec1fd562fe629b389d86603915448078b8fd7e631c8fc9a7d126eb889a1ba0c17611369b190
+
+PKG_BUILDDIR ?= $(PKGDIRBASE)/$(PKG_NAME)
+PKG_ZIPFILE = $(PKGDIRBASE)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT)
+
+ifneq ($(RIOTBASE),)
+include $(RIOTBASE)/Makefile.base
+endif
+
+.PHONY: all clean distclean
+
+prepare: $(PKG_BUILDDIR)/
+
+all: $(PKG_BUILDDIR)/
+	"$(MAKE)" -C $(PKG_BUILDDIR)/src -f $(CURDIR)/Makefile.c25519
+
+$(PKG_BUILDDIR)/: $(PKGDIRBASE)/$(PKG_NAME)-$(PKG_VERSION).$(PKG_EXT)
+	test "$(PKG_SHA512)  $(PKG_ZIPFILE)" =  "$$(sha512sum "${PKG_ZIPFILE}")"
```

Comment:
Question from the sidelines: Why don't you put the hash into a file so you can just use sha512sum?

Reply:
That was what I had at [this point](https://github.com/RIOT-OS/RIOT/commit/9430b2b2d29af173b6f6e75c0412639c9998b9de#commitcomment-33032667), but @kaspar030 requested me to change it to `test`

Reply:
there are two extra spaces in that line now, the first should actually break the comprison.

> Why don't you put the hash into a file so you can just use sha512sum?

Hm, to me it seemed easier (less lines, less system calls, less files created, less shell invokations) this way.

Reply:
Ah, sorry for the noise. Wasn't following the discussion very well

Reply:
> there are two extra spaces in that line now, the first should actually break the comprison.

scratch that please, good as is.